Modules/TestQuestionPool/classes/class.ilAssExcelFormatHelper.php

```diff
@@ -70,12 +69,11 @@ public function setCellByCoordinates($a_coords, $a_value): void
     public function setCell($a_row, $a_col, $a_value, $datatype = null): void
     {
         if (is_string($a_value) && !is_numeric($a_value)) {
-            $this->workbook->getActiveSheet()->setCellValueExplicitByColumnAndRow(
-                $a_col + 1,
-                $a_row,
+            $coordinate = $this->getCoordByColumnAndRow($a_col, $a_row);
```

Comment:
The `+1` on the column index is applied by a call to `columnIndexAdjustment` in `getCoordByColumnAndRow`.